datalake/src/main/java/com/sequenceiq/datalake/service/upgrade/ccm/SdxCcmUpgradeService.java

```diff
@@ -69,24 +78,34 @@ public SdxCcmUpgradeResponse upgradeCcm(String environmentCrn) {
         }
     }
 
+    public void initAndWaitForStackUpgrade(SdxCluster sdxCluster, PollingConfig pollingConfig) {
+        String stackCrn = sdxCluster.getStackCrn();
+        LOGGER.debug("Initiating CCM upgrade on stack CRN {} for datalake {}", stackCrn, sdxCluster.getName());
+        // String initiatorUserCrn = ThreadBasedUserCrnProvider.getUserCrn();
```

Comment:
Can you delete this comment?

Reply:
it will be the next phase so it will be uncommented eventually